packages/design-picker/src/hooks/use-theme-designs-query.ts

```diff
@@ -53,20 +53,17 @@ function apiThemeToDesign( { id, name, taxonomies, stylesheet, price, theme_tier
 		( { slug }: any ) => slug === 'featured'
 	);
 
-	const is_premium = stylesheet && stylesheet.startsWith( 'premium/' );
-
 	return {
 		slug: id,
 		title: name,
 		recipe: {
 			stylesheet,
 		},
-		is_premium,
 		categories: taxonomies?.theme_subject ?? [],
 		is_featured_picks: isFeaturedPicks,
 		showFirst: isFeaturedPicks,
 		...( STATIC_PREVIEWS.includes( id ) && { preview: 'static' } ),
-		design_type: is_premium ? 'premium' : 'standard',
+		design_type: theme_tier?.slug === 'premium' ? 'premium' : 'standard',
```

Comment:
I think we need the tier slug to be either `Premium` or `Personal` themes for this to evaluate to `'premium'` right? Is the tier we get here the same tier we get from themes? If not I would rename this to something else to avoid confusion. There are also other places in this PR where this would apply.

Reply:
Yes, tier there is the same tier as we get from themes. We call them Designs in the design picker part of calypso because sometimes they're themes, sometimes they're virtual themes, sometimes it's assembler and in the past there's been other stuff.

All that this prop is ever used for realistically is to differentiate between assembler and everything else. I'll update it to use 'free' to mean 'standard' and everything else to mean 'premium' though.

I'll double check the others, but by and large `design_type` and `is_premium` have been misleading at best when used to differentiate between free, paid and subscribe options, so I've just retained the existing logic. People _should_ be using tiers for analytics, not these properties.